### ISSUE TITLE
Option to turn off the hash value before commit

### ISSUE THREAD

**Issue by johnny-bui** [nicoespeon/gitgraph.js#45]
How can I turn off the hash value before every commit (extended mode)?

**Comment by ddidier** [comment]
See #53

**Comment by johnny-bui** [comment]
Thank you,
I'll use it for my Uni project.  :)

**Comment by nicoespeon** [comment]
Issue fixed with #53 — thanks a lot @ddidier 

You can now turn off hash value using options:

```javascript
// Specific to a commit
graph.commit( { messageHashDisplay: false } );

// Applied on your own custom template
var myTemplateConfiguration = {
  commit: {
    message: { displayHash: false }
  }
};
var myTemplate = new GitGraphTemplate( myTemplateConfig );
var graph = new GitGraph( { template: myTemplate } );
```
